spec/teacher_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../teacher'
+
+describe Teacher do
```

Comment:
- [ ] I suggest you test the `can_use_services?` method in the teacher.rb class since all the methods in all the classes should be tested :blush: